packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -564,59 +564,53 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
     }
 
     private async fetchSnapshot(snapshotOptions: ISnapshotOptions, tokenFetchOptions: TokenFetchOptions) {
+        assert(snapshotOptions.timeout !== undefined, "Timeout should be provided for setting a limit");
```

Comment:
Not sure if it's appropriate here (could result in explosion of types, and also can be clunky to use by callers), but consider creating new interface where timeout property is not optional, and thus have it compiler check at compilation time, then detecting bugs at runtime via asserts

Reply:
Actually this is private api and we are definitely adding timeout by ourselves, so I can remove this assert with any issue. So removing it.